addons/point_of_sale/models/pos_session.py

```diff
@@ -25,6 +25,12 @@ class PosSession(models.Model):
         ('closed', 'Closed & Posted'),
     ]
 
+    POS_PRODUCT_PRODUCT_FIELDS = [
+        'display_name', 'lst_price', 'standard_price', 'categ_id', 'pos_categ_id', 'taxes_id', 'barcode',
+        'default_code', 'to_weight', 'uom_id', 'description_sale', 'description', 'product_tmpl_id', 'tracking',
+        'available_in_pos', 'attribute_line_ids', 'active', '__last_update', 'image_128'
+    ]
+
```

Comment:
I believe it's better to create a function for this, especially since it is intended to be overridden and customized as needed.

Reply:
I don't see any evidence that "it is intended to be overridden", what make you think that it is the case ?

Reply:
Other modules have the capability to introduce new fields to the product, and they can choose to override your function for adding these fields instead of relying on '_loader_params_product_product'.

Reply:
nice catch! I'll adapt this in the next amended commit :-)